n2y/blocks.py

```diff
@@ -468,8 +468,13 @@ class LinkPreviewBlock(WarningBlock):
     pass
 
 
-class SyncedBlock(WarningBlock):
-    pass
+class SyncedBlock(Block):
+    def to_pandoc(self):
+        synced_block_shared = self.has_children
+        if not synced_block_shared:
+            logger.warning('Skipping un-shared synced block (%s)', self.notion_url)
```

Comment:
👍 this should definitely be a warning; nice job including the url; it makes it easy to track down and fix this stuff